test-it/karma/fixtures/publicapi/AllocateTypes.via

```diff
@@ -0,0 +1,57 @@
+// Globals
+define(GlobalInt32 Int32)
+define(GlobalString String)
+define(GlobalErrorCluster ErrorCluster)
+define(GlobalClusterOfScalars c(
+    e(.Boolean bool)
+    e(.String string)
+    e(.Double double)
+    e(.Int32 int32)
+    e(.Int64 int64)
+    e(.ComplexDouble complex)
+    e(.Timestamp time)
+))
+
+define(GlobalArrayOfClusters a(c(
+    e(.Boolean bool)
+    e(.String string)
+    e(.Double double)
+    e(.Int32 int32)
+    e(.Int64 int64)
+    e(.ComplexDouble complex)
+    e(.Timestamp time)
+) *))
+
+define(GlobalClusterOfArrays c(
+    e(a(.Boolean *) booleans)
+    e(a(.String *) strings)
+    e(a(.Double *) doubles)
+    e(a(.Int32 *) int32s)
+    e(a(.Int64 *) int64s)
+    e(a(.UInt64 *) uint64s)
+    e(a(.ComplexDouble *) complexes)
+    e(a(.Timestamp *) times)
+))
+
+define(AllocateTypes dv(.VirtualInstrument (
+    Locals: c(
+        e(a(.Boolean *) booleans)
+        e(a(.String *) strings)
+        e(a(.Double *) doubles)
+        e(a(.Int32 *) int32s)
+        e(a(.Int64 *) int64s)
+        e(a(.UInt64 *) uint64s)
+        e(a(.ComplexDouble *) complexes)
+        e(a(.Timestamp *) times)
+
+        e(a(.Boolean 2 2) fixedBooleans)
+        e(.AnalogWaveform<.Double> wave_Double)
+        e(.NIPath nipath)
+        e(dv(Enum16 (zero one two three four) 3) enum16numbers)
+    )
+    clump (1
+    )
+)))
+
```

Comment:
It is a good idea to test globals. We could add a few tests for "ce" in "Locals" as that's what DFIR emits now for constants. It is probably a good idea to add tests for multi-dimensional arrays but can be done later and this need not be blocked.

Reply:
I added a few cases for `ce` in locals and a 3D array of single values.